### PR TITLE
creates a gugasanchez personal page

### PR DESCRIPTION
**Pull request by gugasanchez**
## Description

- Created a personal page including photo, name, short bio, address and some links to social network
- Included function to let user copy my address if needed
- Tried to make it using the best practices of frontend and according to SE2 other components

image

## Additional Information

- [X] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [X] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

Your ENS/address: 0xeA2f0BB0144fbff64E6f6F492B999674f9393882

### CODE REVIEW

File: packages/nextjs/app/builders/0xeA2f0BB0144fbff64E6f6F492B999674f9393882/page.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+          </a>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default GustavosPage;
+
+// now, in this code, i want to include a div flex row in which the logos of github and linkedin and telegram appear with w-30 and h-30 each that, when clicked, send to my github profile and my linkedin and my telegram profiles.
```
Comment (carletex) [inline packages/nextjs/app/builders/0xeA2f0BB0144fbff64E6f6F492B999674f9393882/page.tsx:121]: Copilot prompt? :D

Let's remove it.

File: packages/nextjs/app/builders/0xeA2f0BB0144fbff64E6f6F492B999674f9393882/page.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            <CopyToClipboard text="0xeA2f0BB0144fbff64E6f6F492B999674f9393882" onCopy={() => setAddressCopied(false)}>
+              <div className="dropdown-content absolute top-9 left-0 px-4 bg-white border border-gray-200 rounded-md shadow-lg z-10 cursor-pointer">
+                <div className="flex items-center py-2 space-x-2">
+                  <CheckCircleIcon className="h-6 w-6 text-green-500" aria-hidden="true" />
+                  <span>Copy address</span>
+                </div>
+              </div>
+            </CopyToClipboard>
+          )}
+        </div>
```
Comment (carletex) [inline packages/nextjs/app/builders/0xeA2f0BB0144fbff64E6f6F492B999674f9393882/page.tsx:87]: I appreciate you doing this, but this adds a lot of bloat. (Thinking if this were going to be a contribution for SE-2)

```
<Address address="0xeA2f0BB0144fbff64E6f6F492B999674f9393882" />
```

Will have almost the same effect, and we want to reuse & simplify as much as we can.

**Review by carletex — CHANGES_REQUESTED** [review]
This looks great Gustavo!!

Just left a couple of comments (for the sake of simplicity)

Thanks!

File: packages/nextjs/app/builders/0xeA2f0BB0144fbff64E6f6F492B999674f9393882/page.tsx (hunk 2 above)
Comment (gugasanchez) [reply packages/nextjs/app/builders/0xeA2f0BB0144fbff64E6f6F492B999674f9393882/page.tsx:87]: Perfect, Carlos! Working on it right now!

Thank you very much. :)

**Comment by gugasanchez** [comment]
@carletex just made the changes you proposed! Thanks a lot once again.

**Review by carletex — APPROVED** [review]
Looking great!

I removed the "use client" since it's not needed now. More info here: https://github.com/BuidlGuidl/batch4.buidlguidl.com/pull/13#discussion_r1557574281

**Pull request merged**